assets/javascripts/openqa.js

```diff
@@ -261,13 +272,12 @@ function restartJob(ajaxUrl, jobId) {
       if (newJobUrl) {
         window.location.replace(newJobUrl);
       } else {
-        showError('URL for new job not available');
+        throw 'URL for new job not available';
```

Comment:
is there any reason to not stay with `showError` here?

Reply:
We actually do.
The throw will jump to the catch.
And the catch executes `showError`.